admin/main.go

```diff
@@ -184,6 +185,178 @@ func verifyIDToken(app *firebase.App, idToken string) *auth.Token {
+	return u
+}
+
+func getUserByPhone(ctx context.Context, client *auth.Client) *auth.UserRecord {
+	phone := "+13214567890"
+	// [START get_user_by_phone]
+	u, err := client.GetUserByPhoneNumber(ctx, phone)
+	if err != nil {
+		log.Fatalf("error getting user by phone %s: %v\n", phone, err)
+	}
+	log.Printf("Successfully fetched user data: %v\n", u)
+	// [END get_user_by_phone]
+	return u
+}
+
+func createUser(ctx context.Context, client *auth.Client) *auth.UserRecord {
+	// [START create_user]
+	u, err := client.CreateUser(context.Background(),
+		(&auth.UserToCreate{}).
+			Email("user@example.com").
+			EmailVerified(false).
+			PhoneNumber("+15555550100").
+			Password("secretPassword").
+			DisplayName("John Doe").
+			PhotoURL("http://www.example.com/12345678/photo.png").
+			Disabled(false))
+
+	if err != nil {
+		log.Fatalf("error creating user: %v\n", err)
+	}
+	log.Printf("Successfully created user: %v\n", u)
+	// [END create_user]
+	return u
+}
+
+func createUserWUID(ctx context.Context, client *auth.Client) *auth.UserRecord {
+	uid := "something"
+	// [START create_user_with_uid]
+	u, err := client.CreateUser(context.Background(),
+		(&auth.UserToCreate{}).UID(uid).Email("user@example.com").PhoneNumber("+15555550100"))
+	// Alternatively
+	params := auth.UserToCreate{} // also possible: 	var params auth.UserToCreate
+	u2, err := client.CreateUser(context.Background(),
+		params.UID(uid).Email("user@example.com").PhoneNumber("+15555550100"))
+
+	if err != nil {
+		log.Fatalf("error creating user: %v\n", err)
+	}
+	log.Printf("Successfully created user: %v\n", u)
+	// [END create_user_with_uid]
+	return u2
+}
+
+func updateUser(ctx context.Context, client *auth.Client) (*auth.UserRecord, *auth.UserRecord) {
+	uid := "d"
+	// [START update_user]
+	u, err := client.UpdateUser(context.Background(), uid,
+		(&auth.UserToUpdate{}).
+			Email("user@example.com").
+			EmailVerified(true).
+			PhoneNumber("+15555550100").
+			Password("newPassword").
+			DisplayName("John Doe").
+			PhotoURL("http://www.example.com/12345678/photo.png").
+			Disabled(true))
+	// Alternatively
+	params := auth.UserToUpdate{} // also possible: 	var params auth.UserToUpdate
+	u2, err := client.UpdateUser(context.Background(), uid,
+		params.Email("user@example.com").PhoneNumber("+15555550100"))
+
+	if err != nil {
+		log.Fatalf("error updating user: %v\n", err)
+	}
+	log.Printf("Successfully updated user: %v\n", u)
+	// [END update_user]
+	return u, u2
+}
+
+func deleteUser(ctx context.Context, client *auth.Client) {
+	uid := "d"
+	// [START delete_user]
+	err := client.DeleteUser(context.Background(), uid)
+	if err != nil {
+		log.Fatalf("error deleting user: %v\n", err)
+	}
+	log.Printf("Successfully deleted user: %s\n", uid)
+	// [END delete_user]
+}
+
+func customClaims(ctx context.Context, client *auth.Client) {
+	uid := "uid"
+	// [START custom_claims]
+
+	// erase all existing custom claims
+	err := client.SetCustomUserClaims(context.Background(), uid, nil)
```

Comment:
Please remove. The snippet here should be equivalent to the Node.js snippet in the same page.